packages/containers/src/HomeListView/HomeListView.component.js

```diff
@@ -1,16 +1,13 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { Layout } from '@talend/react-components';
+import { Inject } from '@talend/react-cmf';
 
-import AppHeaderBar from '../AppHeaderBar';
-import List from '../List';
-import SidePanel from '../SidePanel';
-
-function getContent(Component, props) {
+function getContent(component, props) {
```

Comment:
please add jsdoc, wich describe
for example component can be either : a component, a string, null, undefined
Should we add a deprecation warning on `if (React.isValidElement(props)) {` block